package.json

```diff
@@ -26,13 +26,14 @@
   "types": "./lib/main/ts/api/Main.d.ts",
   "dependencies": {
     "@types/escodegen": "0.0.6",
-    "@types/estree": "0.0.39",
-    "@types/node": "^12.7.11",
+    "@types/estree": "0.0.45",
+    "@types/node": "*",
```

Comment:
Is this meant to be `*`?

Reply:
Yes, it is meant to be. As it can run on multiple NodeJS versions.

Reply:
Could we put `minVersion - maxVersion` rather than saying that we work with _all_ node versions?